jax/_src/numpy/lax_numpy.py

```diff
@@ -467,16 +467,16 @@ def result_type(*args: Any) -> DType:
 
 @jit
 def trunc(x: ArrayLike) -> Array:
-  """Return the element-wise truncated value of the input.
+  """Return truncated value of the input to the nearest integer towards zero.
 
   JAX implementation of :func:`numpy.trunc`.
 
   Args:
     x: input array or scalar.
 
   Returns:
-    An array containing the truncated values of ``x``. The truncated values of the
-    elements of ``x`` are closer to ``0``, than the elements themselves.
+    An array with same shape and dtype as ``x``, containing the values of
+    ``x`` truncated to the nearest integer towards ``0``.
```

Comment:
You could use the same `Returns` description here as in `fix` below, it's much more clear and concise.